server/models.py

```diff
@@ -151,17 +151,27 @@ class Relation(models.TextChoices):
 
 
 class Event(ExportModelOperationsMixin("event"), models.Model):  # type: ignore[misc]
+    class Type(models.TextChoices):
+        MIXED = "MXD", _("Mixed")
+        OPENS = "OPN", _("Opens")
+        WOMENS = "WMN", _("Womens")
+
     title = models.CharField(max_length=255)
     start_date = models.DateField()
     end_date = models.DateField()
+    registration_start_date = models.DateField(null=True, blank=True)
+    registration_end_date = models.DateField(null=True, blank=True)
     ultimate_central_id = models.PositiveIntegerField(unique=True, null=True, blank=True)
     ultimate_central_slug = models.SlugField(max_length=200, default="unknown")
     location = models.CharField(max_length=255, default="unknown")
+    type = models.CharField(max_length=3, choices=Type.choices, default=Type.MIXED)
```

Comment:
Are we showing this in UI now? Be good to show atleast in All tournaments and reg page for sakkath? cause we have opens and womens